### ISSUE TITLE
[Bug report]: Firefly 2.0.1 for Windows does not start - white screen appears after Firefly splash screen

### ISSUE THREAD

**Issue by PK-66** [iotaledger/firefly#7546]
### Platform

Desktop

### Version

2.0.1

### Description

Hi, 
I installed the latest version of Firefly for IOTA (2.0.1). Once installed, Firefly began to start. At first it looked like it was working as it came up with the Splash screen saying Firefly. But then the Firefly writing disappeared and I was left with a blank white screen where it just hung constantly. Eventually I just closed the window by killing the process. I tried starting Firefly again a few times but with the same results. 
As I couldn't even get to the page that shows all my Profiles, I decided to uninstall Firefly and re-install version 1.7.4, which is the one I was on before. However, after installation, I do get the Profiles window, but after I have entered my 6 digit passcode, yet again the screen goes white and just hangs again!
Please can you help.

### Expected behaviour

When I start the application I expect to see the Splash window, followed by the screen showing my list of Profiles.

### Actual behaviour

Once installed, Firefly began to start. At first it looked like it was working as it came up with the Splash screen saying Firefly. But then the Firefly writing disappeared and I was left with a blank white screen where it just hung constantly. Eventually I just closed the window by killing the process. I tried starting Firefly again a few times but with the same results. 

### Can the issue reliably be reproduced?

Yes

### Operating System

Windows

### Diagnostics

```shell
It doesn't even get into Firefly so I can't get to the Diagnostics.
```

### Developer Profile

No

### Profile Type

Software (Stronghold)

### Staking

Previously staked

### Crash Status

White screen

### Error Messages

None

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by CapnGerry** [comment]
I'm geting the same issue.

**Comment by CapnGerry** [comment]
Tried uninstalling and installing older 1.7.0 version. It gets as far as allowing me to input PIN No then goes white. Mon, 21 Feb 2022 16:54:26 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 08 Feb 2022 14:38:32 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 08 Feb 2022 14:24:53 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 08 Feb 2022 14:18:59 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 08 Feb 2022 13:45:43 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 08 Feb 2022 13:45:33 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 08 Feb 2022 13:45:22 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 08 Feb 2022 13:18:19 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 08 Feb 2022 13:17:49 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Sun, 06 Feb 2022 17:28:51 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 04 Feb 2022 16:18:12 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 04 Feb 2022 16:15:32 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 04 Feb 2022 16:13:32 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 04 Feb 2022 16:11:34 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 31 Dec 2021 16:41:06 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 31 Dec 2021 16:33:03 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``

Thu, 30 Dec 2021 21:20:00 GMT
ClientError: `error sending request for url (https://chrysalis-nodes.iota.cafe/api/v1/info): operation timed out`

Tue, 28 Dec 2021 13:53:30 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``

Tue, 28 Dec 2021 13:53:30 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``

Mon, 27 Dec 2021 15:52:32 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:52:22 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:51:32 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:51:22 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:50:42 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:49:42 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:49:32 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:45:52 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:45:42 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:44:32 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:41:32 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:41:22 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:40:44 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:39:44 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:39:04 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:36:43 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:36:05 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:32:58 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:31:23 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:25:39 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:23:59 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:22:29 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:20:29 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:19:29 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 27 Dec 2021 15:19:19 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 21 Dec 2021 19:22:46 GMT
ClientError: `error sending request for url (https://chrysalis-nodes.iota.cafe/api/v1/info): operation timed out`

Tue, 21 Dec 2021 19:22:11 GMT
ClientError: `error sending request for url (https://chrysalis-nodes.iota.cafe/api/v1/info): operation timed out`

Tue, 21 Dec 2021 19:21:55 GMT
ReqwestError: error sending request for url (https://chrysalis-nodes.iota.cafe/api/plugins/participation/addresses/iota1qz6vwp7nx8m6dewpe70pthz55hvs45l8pkrkuwna9xe8n47mlsnssaw53ns): error trying to connect: tcp connect error: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060)

Tue, 21 Dec 2021 19:21:46 GMT
ClientError: `error sending request for url (https://chrysalis-nodes.iota.cafe/api/v1/info): operation timed out`

Tue, 21 Dec 2021 19:21:15 GMT
ReqwestError: error sending request for url (https://chrysalis-nodes.iota.cafe/api/plugins/participation/addresses/iota1qz6vwp7nx8m6dewpe70pthz55hvs45l8pkrkuwna9xe8n47mlsnssaw53ns): error trying to connect: tcp connect error: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060)

Tue, 21 Dec 2021 19:19:56 GMT
ClientError: `error sending request for url (https://chrysalis-nodes.iota.cafe/api/v1/info): operation timed out`

Tue, 21 Dec 2021 19:15:20 GMT
ReqwestError: error sending request for url (https://chrysalis-nodes.iota.cafe/api/plugins/participation/addresses/iota1qz82cr7lutztfg0jzlhv2mg9858hct82pv3cxwgjhduc03y6zxc2qf9r903): error trying to connect: tcp connect error: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060)

Tue, 21 Dec 2021 19:14:54 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``

Tue, 21 Dec 2021 19:14:53 GMT
ReqwestError: error sending request for url (https://chrysalis-nodes.iota.cafe/api/plugins/participation/addresses/iota1qz75wl8w9m8v57gwnwm2h9ky72862c2dpppq5n2uxrw4rtnxryxwuw3j2rr): error trying to connect: tcp connect error: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060)

Tue, 21 Dec 2021 19:14:46 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``

Tue, 21 Dec 2021 18:45:37 GMT
ClientError: `error sending request for url (https://chrysalis-nodes.iota.cafe/api/v1/info): operation timed out`

Tue, 21 Dec 2021 18:44:56 GMT
ClientError: `error sending request for url (https://chrysalis-nodes.iota.cafe/api/v1/info): operation timed out`

Sun, 12 Dec 2021 11:46:21 GMT
ClientError: {}

Sun, 12 Dec 2021 11:37:33 GMT
LedgerMiscError: ledger transport error

Sun, 12 Dec 2021 11:01:41 GMT
LedgerMiscError: ledger transport error

Sun, 12 Dec 2021 11:01:30 GMT
ClientError: {}

Sun, 05 Dec 2021 18:01:49 GMT
ClientError: {}

Sun, 05 Dec 2021 17:08:31 GMT
ClientError: {}

Sun, 05 Dec 2021 16:13:47 GMT
ClientError: {}

Sun, 05 Dec 2021 15:33:45 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``

Sun, 05 Dec 2021 15:33:44 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``

Sun, 05 Dec 2021 14:30:37 GMT
ClientError: {}

Mon, 04 Oct 2021 16:06:10 GMT
ClientError: {}

Sun, 01 Aug 2021 14:20:42 GMT
ClientError: {}

Sun, 01 Aug 2021 13:52:27 GMT
ClientError: {}

Wed, 02 Jun 2021 22:29:33 GMT
ClientError: {}

Wed, 02 Jun 2021 22:23:46 GMT
ClientError: {}

Sat, 29 May 2021 13:04:20 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``

Sat, 29 May 2021 12:56:05 GMT
ClientError: {}

Sat, 29 May 2021 12:31:52 GMT
ClientError: {}

Sun, 09 May 2021 22:21:53 GMT
ClientError: {}

Sun, 09 May 2021 20:29:24 GMT
ClientError: {}

Sun, 09 May 2021 20:01:02 GMT
ClientError: {}

Sun, 09 May 2021 14:46:43 GMT
ClientError: {}

**Comment by begonaalvarezd** [comment]
We are working on a fix, will release a new version soon

**Comment by begonaalvarezd** [comment]
We released a new version which fixed the blank screen issue 

### Announcement message

We just released **Firefly IOTA 2.0.2** hotfixing the blank screen and accessing profiles due to error `Chrysalis database migration failed: migration failed no chrysalis data to migrate` :pray: 

:arrow_down:  Download:
https://firefly.iota.org/

:bug:  Bug Reporting:
https://github.com/iotaledger/firefly/issues/new?assignees=&labels=bug+report&template=bug_report.yml&title=%5BBug+report%5D%3A

Happy week to everyone :tulip: and apologies for the troubles :pray:

**Comment by PK-66** [comment]
That's great. I have downloaded the latest version and everything is working fine. Thanks very much!
